website/docs/r/data_protection_backup_policy_kubernetes_cluster.html.markdown

```diff
@@ -0,0 +1,149 @@
+
+* `retention_rule` - (Optional) One or more `retention_rule` blocks as defined below. Changing this forces a new resource to be created.
+
+* `time_zone` - (Optional) Specifies the Time Zone which should be used by the backup schedule. Changing this forces a new resource to be created.
+
+---
+
+A `default_retention_rule` block supports the following:
+
+* `life_cycle` - (Required) A `life_cycle` block as defined below. Changing this forces a new resource to be created.
+
+---
+
+A `retention_rule` block supports the following:
+
+* `name` - (Required) The name which should be used for this retention rule. Changing this forces a new resource to be created.
+
+* `criteria` - (Required) A `criteria` block as defined below. Changing this forces a new resource to be created.
+
+* `life_cycle` - (Required) A `life_cycle` block as defined below. Changing this forces a new resource to be created.
+
+* `priority` - (Required) Specifies the priority of the rule. The priority number must be unique for each rule. The lower the priority number, the higher the priority of the rule. Changing this forces a new resource to be created.
+
+---
+
+A `criteria` block supports the following:
+
+* `absolute_criteria` - (Optional) Possible values are `AllBackup`, `FirstOfDay`, `FirstOfWeek`, `FirstOfMonth` and `FirstOfYear`. These values mean the first successful backup of the day/week/month/year. Changing this forces a new resource to be created.
+
+* `days_of_week` - (Optional) Possible values are `Monday`, `Tuesday`, `Thursday`, `Friday`, `Saturday` and `Sunday`. Changing this forces a new resource to be created.
+
+* `months_of_year` - (Optional) Possible values are `January`, `February`, `March`, `April`, `May`, `June`, `July`, `August`, `September`, `October`, `November` and `December`. Changing this forces a new resource to be created.
+
+* `scheduled_backup_times` - (Optional) Specifies a list of backup times for backup in the `RFC3339` format. Changing this forces a new resource to be created.
+
+* `weeks_of_month` - (Optional) Possible values are `First`, `Second`, `Third`, `Fourth` and `Last`. Changing this forces a new resource to be created.
+
+---
+
+A `life_cycle` block supports the following:
+
+* `data_store_type` - (Required) The type of data store. Possible values is `OperationalStore`. 
+
+* `duration` - (Required) The retention duration up to which the backups are to be retained in the data stores. It should follow `ISO 8601` duration format. Changing this forces a new resource to be created.
+
+* `target_copy_setting` - (Optional) A `target_copy_setting` block as defined below. Changing this forces a new resource to be created.
+
+---
+
+A `target_copy_setting` block supports the following:
+
+* `copy_after` - (Required) Specifies when the backups are tiered across two or more selected data stores as a json encoded string. Changing this forces a new resource to be created.
```

Comment:
since there is only one type of `data_store_type`. Do there exist a real scenario that `the backups are tiered across two or more selected data stores`?

Reply:
Thank you for your question. Yes, this has been confirmed with the services team, but is currently in preview (feature is already supported in Azure Portal). Considering that the service team did not request onboard this preview feature this time, I deleted `target_copy_setting` in the code, set to empty array.